tsconfig.base.json

```diff
@@ -7,14 +7,14 @@
     "target": "es2020",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
Not sure why this is diffing so weirdly. Sigh. Anyone know?

Reply:
This is because JSON technically aren't allowed to have comments, so GH is higlighting the syntax errors (even for the parts of the files that didn't change).